.github/CONTRIBUTING.md

```diff
@@ -29,34 +29,27 @@ In these cases, please link the issues across repositories (e.g. add `will be im
 Some of the related software we will be using and contributing to.
 
 * BETYdb: github.com/pecanproject/bety
-* CoGe (genomics pipeline): https://github.com/LyonsLab/coge
-* NCO (raster data processing): https://github.com/nco/nco
-* Clowder: https://opensource.ncsa.illinois.edu/bitbucket/projects/CATS/repos/clowder/browse
 * Breeding Management Systems
 * PlantCV (image processing): https://github.com/danforthcenter/plantcv
 
 ## Creating Issues
 
 - Make sure you have a GitHub account.
 - Search GitHub and Google to see if your issue has already been reported
-        - Create an issue in GitHub, assuming one does not already exist.
+    - Create an issue in GitHub, assuming one does not already exist.
 	- Clearly describe the issue including steps to reproduce when it is a bug.
 	- Make sure you fill in the earliest version that you know has the issue.
-- Ask @dlebauer or @robkooper to add you to the TERRA Ref project if you plan on fixing the issue.
-
-* Github Issues
-  * [computing pipeline (infrastructure, algorithms)](github.com/terraref/computing-pipeline/issues/new)
-  * [reference data (data products)](github.com/terraref/reference-data/issues/new)
+- Ask @dlebauer or @chris-schnaufer to add you to the AgPipeline project if you plan on fixing the issue.
```

Comment:
is this necessary within a fork + PR workflow? I thought that one of the advantages of using forks is that people don't need to be added to the organization ...

Reply:
Changed to reflect this and added more links to our OSF page on Issues and Pull Requests: https://osf.io/tzmhp/wiki/Issues%20and%20Pull%20Requests/

Reply:
Good catch: not needed